Sources/StreamChatUI/CommonViews/ChatInputView/ChatCommandLabel.swift

```diff
@@ -5,44 +5,43 @@
 import StreamChat
 import UIKit
 
-public typealias ChatMessageInputSlashCommandView = _ChatMessageInputSlashCommandView<NoExtraData>
+/// A view that display the command name and icon.
+public typealias ChatCommandLabel = _ChatCommandLabel<NoExtraData>
 
-open class _ChatMessageInputSlashCommandView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
-    // MARK: - Properties
-    
-    public var commandName: String? {
+/// A view that display the command name and icon.
+open class _ChatCommandLabel<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
+    /// The command that the label displays.
+    public var content: Command? {
         didSet {
             updateContentIfNeeded()
         }
     }
 
-    // MARK: - Subviews
-    
-    public private(set) lazy var container = ContainerStackView()
+    /// The container stack view that layouts the label and the icon view.
+    public private(set) lazy var container: ContainerStackView = ContainerStackView()
```

Comment:
Can we let the compiler do its job? 😄
```swift
public private(set) lazy var container = ContainerStackView()
```

Reply:
This is part of the pattern with Audit of the components, that each subview should have its type explicit. I guess this is more important on the views that come from the `UIConfig` but to follow the same pattern, we do it everywhere if I'm not mistaken, right? CC @VojtaStavik Probably this is missing from the Audit Document, I'll add it

Reply:
Yes, we try to make it easier for the user by specifying the type explicitly. As @nuno-vieira said, for UIConfig views, it's important much more, here it doesn't matter that much :)